src/IdeaRS.OpenModel/Libraries/Material/BoltAssembly.cs

```diff
@@ -127,5 +127,19 @@ public bool WasherAtNut
 		}
 
 		#endregion
+
+		/// <summary>
+		/// Length
+		/// </summary>
+		public double Length
```

Comment:
This property should be on the BoltGrid?

Reply:
removed